datafusion/core/tests/parquet/row_group_pruning.rs

```diff
@@ -447,17 +447,15 @@ macro_rules! int_tests {
     };
 }
 
-int_tests!(8, correct_bloom_filters: false);
-int_tests!(16, correct_bloom_filters: false);
-int_tests!(32, correct_bloom_filters: true);
-int_tests!(64, correct_bloom_filters: true);
+int_tests!(32);
```

Comment:
I think it would help to give future readers a hint here about why 8/16 aren't supported (rather than just an oversight):

```suggestion
// int8/int16 are incorrect: https://github.com/apache/arrow-datafusion/issues/9779
int_tests!(32);
```